src/typegen.ts

```diff
@@ -16,14 +17,31 @@ export function generate(options: Options): Promise<void> {
   return doGenerate(false, options)
 }
 
+function isModule(name: string) {
+  return (
+    !name.includes('/') ||
+    (name.startsWith('@') && (name.match(/\//g) || []).length === 1)
+  )
+}
+
 export function doGenerate(sync: true, options: Options): void
 export function doGenerate(sync: false, options: Options): Promise<void>
 export function doGenerate(
   sync: boolean,
   options: Options,
 ): void | Promise<void> {
   const dmmf = DMMF.get(options.photonPath)
-  const tsDeclaration = render(dmmf, options.photonPath)
+  console.log({ options })
+  const tsDeclaration = render(
+    dmmf,
+    isModule(options.photonPath)
+      ? options.photonPath
+      : // Support Serverless platforms by resolving module paths relatively
+        './' +
+          path
+            .relative(path.dirname(options.typegenPath), options.photonPath)
+            .replace(/\\/g, '/'), // Use '/' instead of '\' on Windows
```

Comment:
A windows util func might be warranted here, like you did for `quoteOnWindows`. Would be re-used in the test suite too.

Reply:
Thank you for the suggestion! Unfortunately, I don’t have access to a computer right now, but feel free to modify the PR.